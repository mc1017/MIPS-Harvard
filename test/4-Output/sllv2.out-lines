CPU : OUT :2149056512
