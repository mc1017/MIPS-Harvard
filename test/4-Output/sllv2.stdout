CPU : OUT :2149056512
TB : Finished : active=0
test/1-ArithmeticandLS/sllv2.v:38: $finish called at 44 (1s)
